package.json

```diff
@@ -18,10 +18,12 @@
     "start:dev": "./scripts/start-dev.sh",
     "start:dev:remote": "npm run setup:dev:remote && DATA_SOURCE=remote npm run start:dev",
     "start:dev:mock": "DATA_SOURCE=mock npm run start:dev",
-    "test": "$(npm bin)/jest --coverage",
     "test:watch": "npm run test -- --watch",
```

Comment:
The `--watch` param doesn't make it through to jest (this runs `npm run test:unit --watch`, which doesn't have the `--` separator). Basically when we use `-- --foo` the script we're calling can't be another `npm run`. We should remove the existing `test:watch` script and add these:

```json
"test:unit:watch": "npm run test:unit -- --watch",
"test:watch": "npm run test:unit:watch",
```

Reply:
Thanks for highlighting this.
I think we just need this:
```
"test": "npm run test:unit -- --coverage",
"test:unit": "$(npm bin)/jest --rootDir=. --config=config/jest.config.js",
"test:watch": "npm run test:unit -- --watch",
```